frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Openverse frontend",
   "private": true,
   "version": "2.2.1",
+  "packageManager": "pnpm@9.5.0",
```

Comment:
Please correct me if I'm wrong but I was under the impression that `packageManager` is only read from the `package.json` in the repo root.

Reply:
This is necessary for the Playwright Docker setup, not the frontend (I think frontend can use the root package.json). The [node corepack docs](https://nodejs.org/api/corepack.html#corepack_configuring_a_package) say "_The Corepack proxies will find the **closest** [package.json](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions) file in your current directory hierarchy to extract its ["packageManager"](https://nodejs.org/api/packages.html#packagemanager) property._"